### PR TITLE
KAZOO-4359 restore filtered empty value

### PR DESCRIPTION
**Pull request by jamhed**

### CODE REVIEW

File: core/kazoo/src/api/kapi_sysconf.erl [hunk 1]
```diff
@@ -129,6 +129,13 @@ set_req(Prop) when is_list(Prop) ->
 set_req(JObj) ->
     set_req(kz_json:to_proplist(JObj)).
 
+-spec fix_req(kz_props:props(), api_terms(), function()) -> {'ok', iolist()} | {'error', string()}.
+fix_req(Api, Props, Fun) ->
+    case {props:get_value(<<"Key">>, Api), props:get_value(<<"Value">>, Api)} of
+        {<<"fs_nodes">>, []} -> Fun([{<<"Value">>, []} | Props]);
```
Comment (lazedo) [inline core/kazoo/src/api/kapi_sysconf.erl]: `kapi_sysconf`is not about `fs_nodes`. this is not a valid fix

Comment (jamhed) [inline core/kazoo/src/api/kapi_sysconf.erl]: @lazedo, why does it fix the issue then?

Comment (lazedo) [inline core/kazoo/src/api/kapi_sysconf.erl]: you are fixing the effect and creating an exception for something outside the scope of this app.

**Comment by lazedo** [comment]
@jamhed same here

**Pull request closed without merging**